modules/Layout/Notifications.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { ReactElement } from 'react';
+import { Notification } from '@prezly/sdk';
+
+import styles from './Notifications.module.scss';
+import classNames from 'classnames';
+
+interface Props {
+    notifications: Notification[];
+}
+
+export default function Notifications({ notifications }: Props) {
```

Comment:
@oleg-semyonov 
We'll need the exact same `Notifications` components in all the themes. Do you think it makes sense to share the same implementation via the theme-kit package?

Reply:
Yeah, it makes sense to extract it. Although the theme-kit doesn't enforce any styling (it only compiles JS), so if this components is mostly about displaying the stuff, it should either remain in the theme, or be moved to `themes-ui-components` package.